doc/examples/plot_registration.py

```diff
@@ -36,7 +36,7 @@
 
 y, x = np.meshgrid(np.arange(nl), np.arange(nc), indexing='ij')
 
-wI1 = warp(I1, np.array([y+v, x+u]), mode='nearest')
+wI1 = warp(I1, np.array([y+v, x+u]), mode='constant')
```

Comment:
```suggestion
wI1 = warp(I1, np.array([y+v, x+u]), mode='edge')
```
Thank you for spotting this :wink:. Instead of the `constant` mode, I would recommend the `edge` mode: the `constant` mode will in most cases introduce high gradient values on the image border, and thus create artifacts :confused: